dts/bindings/timer/andestech,atcpit100.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2022, Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    This is a representation of the Andes Technology atcpit100 PIT node
+
+compatible: "andestech,atcpit100"
+
+include: base.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    clock-frequency:
+      type: int
+      description: channel clock source
+      required: true
+
+    prescaler:
+      type: int
+      default: 1
+      description: |
+        The prescaler value defines the counter frequency
+        (clock-frequency/prescaler) in atcpit100 counter driver, the prescaler
+        value could be in range [1 .. clock-frequency].
+
+        Defaults to 1 to use the PIT clock-frequency as the counter frequency.
```

Comment:
I assume a value of 1 is effect no prescaler. (if so, please clarify that here).

Reply:
Yes, it means no prescaler, I have updated the description , thank you.